meson.build

```diff
@@ -0,0 +1,53 @@
+project(
+  'pyperfdump',
+  'cpp',
+  version: '1.1',
+  meson_version: '>=1.6.1',
+  default_options: ['buildtype=release',
+                    'python.install_env=auto'],
+)
+
+project_description = 'Python Performance Dump module for PAPI'
+
+pyperfdump_headers = ['papi_utils.h', 'pyperfdump.h']
+pyperfdump_sources = ['src/dump_functions.cpp',
+                      'src/papi_utils.cpp',
+                      'src/perf_dump.cpp']
+inc = include_directories('include')
+
+py_dep = dependency('python3', required: true)
```

Comment:
This is technically unneeded.

https://mesonbuild.com/Python-module.html#extension_module
> *since 0.63.0* `extension_module` automatically adds a dependency to the library if one is not explicitly provided. To support older versions, the user may need to add `dependencies: py_installation.dependency()`, see [`dependency()`](https://mesonbuild.com/Reference-manual_functions.html#dependency).

You require 1.6.1 anyway.